src/webrtc/stats/statsReport.ts

```diff
@@ -28,10 +28,14 @@ export type TrackID = string;
 export type ByteSend = number;
 
 export interface ByteSentStatsReport extends Map<TrackID, ByteSend> {
+    callId: string;
+    opponentMemberId: string;
     // is a map: `local trackID` => byte send
 }
 
 export interface ConnectionStatsReport {
+    callId: string;
+    opponentMemberId: string;
     bandwidth: ConnectionStatsBandwidth;
```

Comment:
These are also technically breaking changes

Reply:
How can we avoid these?
It is necessary to mark the reports with the CallId, otherwise we cannot assign them to the call because we distribute them using events.

Reply:
I'd suggest to make all of these new fields and parameters optional, with the implication that if you provide them when calling `build` they'll be set in the in output object